chains/substrate/types.go

```diff
@@ -61,7 +62,10 @@ func (p *proposal) encode() ([]byte, error) {
 func (w *writer) createFungibleProposal(m msg.Message) (*proposal, error) {
 	bigAmt := big.NewInt(0).SetBytes(m.Payload[0].([]byte))
 	amount := types.NewU128(*bigAmt)
-	recipient := types.NewAccountID(m.Payload[1].([]byte))
+	recipient, err := types.NewAccountID(m.Payload[1].([]byte))
+	if err != nil {
```

Comment:
Is it an API change?

Reply:
yes, it's an constructor interface change